lib/screens/study_nurse_screen/configuration_screen/pages/configuration_additional_entries.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+/// Class for additional entries of the configuration pages.
+class ConfigurationAdditionalEntries extends StatefulWidget {
+  /// Corresponding constructor of the class.
+  const ConfigurationAdditionalEntries({
+    required this.callbackAdditionalEntries,
+    Key? key,
+  }) : super(key: key);
+
+  /// Callback function for additional entries.
+  final void Function(String, String) callbackAdditionalEntries;
+
+  @override
+  State<ConfigurationAdditionalEntries> createState() =>
+      _ConfigurationAdditionalEntriesState();
+}
+
+class _ConfigurationAdditionalEntriesState
+    extends State<ConfigurationAdditionalEntries> {
+  @override
+  Widget build(BuildContext context) {
+    return PicosBody(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          PicosLabel(label: AppLocalizations.of(context)!.caseNumber),
+          PicosTextField(
+            hint: AppLocalizations.of(context)!.caseNumber,
+            onChanged: (String? newValue) {
+              widget.callbackAdditionalEntries('entryCaseNumber', newValue!);
+            },
+            validator: (String? value) {
+              if (value == null || value.isEmpty) {
+                return AppLocalizations.of(context)!.enterCaseNumber;
+              }
+              return null;
+            },
+          ),
+          PicosLabel(label: AppLocalizations.of(context)!.patientID),
+          PicosTextField(
+            hint: AppLocalizations.of(context)!.patientID,
+            onChanged: (String? newValue) {
+              widget.callbackAdditionalEntries('entryPatientID', newValue!);
+            },
+            validator: (String? value) {
+              if (value == null || value.isEmpty) {
+                return AppLocalizations.of(context)!.enterPatientID;
+              }
+              return null;
+            },
+          ),
+          PicosLabel(label: AppLocalizations.of(context)!.instituteKey),
+          SizedBox(
+            width: double.infinity,
+            child: PicosSelect(
+              selection: const <String>[
+                '100',
+                '101',
+                '102',
+                '103',
+                '104',
+                '105',
+                '201',
+                '300',
+                '400',
+                '501',
+                '502'
+              ],
+              callBackFunction: (String? value) {
+                widget.callbackAdditionalEntries('entryInstituteKey', value!);
+              },
+              hint: AppLocalizations.of(context)!.instituteKey,
+              validator: (String? value) {
+                if (value == null || value.isEmpty) {
+                  return '''     ${AppLocalizations.of(context)!.enterInstituteKey}''';
+                }
+                return null;
+              },
+            ),
+          ),
```

Comment:
```suggestion
          PicosSelect(
            selection: const <String>[
              '100',
              '101',
              '102',
              '103',
              '104',
              '105',
              '201',
              '300',
              '400',
              '501',
              '502'
            ],
            callBackFunction: (String? value) {
              widget.callbackAdditionalEntries('entryInstituteKey', value!);
            },
            hint: AppLocalizations.of(context)!.instituteKey,
            validator: (String? value) {
              if (value == null || value.isEmpty) {
                return '''     ${AppLocalizations.of(context)!.enterInstituteKey}''';
              }
              return null;
            },
          ),
```
I've tested, that this SizedBox is unnecessary now.